client/src/module/authentication/container/register-container.tsx

```diff
@@ -20,11 +22,22 @@ export default class RegisterContainer extends React.Component<any> {
 		} = AuthStore;
 
 		const submit = () => {
-			register().then((success) => {
+			const rememberUserCredentials = {

```

Comment:
Given all the API calls happening in it, submit function should probably be declared async and return a promise.